vunit/sim_if/nvc.py

```diff
@@ -261,8 +264,7 @@ def simulate(self, output_path, test_suite_name, config, elaborate_only):
 
         if not elaborate_only:
             cmd += ["--no-save"]
-            if self._supports_jit:
-                cmd += ["--jit"]
+            cmd += ["--jit"]
```

Comment:
This implies that VUnit can only work with NVC >= 1.9, and that's why you added the runtimeerror above?

Just willing to clarify, because this feels a second change on top of improving the version detection. Might be worth having two commits, even if we merge them at the same time.

Reply:
Most of the examples worked with earlier releases but there were also many acceptance test failures so it seems better to specify a minimum version that we know is fully working. But I agree it's a logically separate change so I'll do it in a separate PR after this one is merged.